docs/latest/user/egctl.md

```diff
@@ -76,176 +76,1246 @@ spec:
+        endpoints:
+        - lbEndpoints:
+          - endpoint:
+              address:
+                socketAddress:
+                  address: 1.1.1.1
+                  portValue: 3000
+            loadBalancingWeight: 1
+          loadBalancingWeight: 1
+          locality: {}
+      name: default-backend-rule-0-match-0-www.example.com
+      outlierDetection: {}
+      type: STATIC
+- '@type': type.googleapis.com/envoy.admin.v3.ListenersConfigDump
+  staticListeners:
+  - listener:
+      '@type': type.googleapis.com/envoy.config.listener.v3.Listener
+      accessLog:
+      - filter:
+          responseFlagFilter:
+            flags:
+            - NR
+        name: envoy.access_loggers.file
+        typedConfig:
+          '@type': type.googleapis.com/envoy.extensions.access_loggers.file.v3.FileAccessLog
+          path: /dev/stdout
+      address:
+        socketAddress:
+          address: 0.0.0.0
+          portValue: 10080
+      defaultFilterChain:
+        filters:
+        - name: envoy.filters.network.http_connection_manager
+          typedConfig:
+            '@type': type.googleapis.com/envoy.extensions.filters.network.http_connection_manager.v3.HttpConnectionManager
+            accessLog:
+            - name: envoy.access_loggers.file
+              typedConfig:
+                '@type': type.googleapis.com/envoy.extensions.access_loggers.file.v3.FileAccessLog
+                path: /dev/stdout
+            httpFilters:
+            - name: envoy.filters.http.router
+              typedConfig:
+                '@type': type.googleapis.com/envoy.extensions.filters.http.router.v3.Router
+            rds:
+              configSource:
+                apiConfigSource:
+                  apiType: DELTA_GRPC
+                  grpcServices:
+                  - envoyGrpc:
+                      clusterName: xds_cluster
+                  setNodeOnFirstMessageOnly: true
+                  transportApiVersion: V3
+                resourceApiVersion: V3
+              routeConfigName: default-eg-http
+            statPrefix: http
+            upgradeConfigs:
+            - upgradeType: websocket
+            useRemoteAddress: true
+      name: default-eg-http
+- '@type': type.googleapis.com/envoy.admin.v3.RoutesConfigDump
+  staticRouteConfigs:
+  - routeConfig:
+      '@type': type.googleapis.com/envoy.config.route.v3.RouteConfiguration
+      name: default-eg-http
+      virtualHosts:
+      - domains:
+        - '*'
+        name: default-eg-http
+        routes:
+        - match:
+            headers:
+            - name: :authority
+              stringMatch:
+                exact: www.example.com
+            prefix: /
+          route:
+            cluster: default-backend-rule-0-match-0-www.example.com
+```  
+
+In the below example, we will translate the Kubernetes resources (including the Gateway API resources) into xDS
+resources in json format.
 
+```shell
```

Comment:
nit: the `json` example makes me page overwhelming to read, should we just stick to one (`yaml` default) ?